capa/features/insn.py

```diff
@@ -25,20 +25,9 @@ def __init__(self, name: str, description=None):
 
 
 class Property(Feature):
-    def __init__(self, name: str, description=None):
-        super().__init__(name, description=description)
-
-
-class ReadProperty(Property):
-    def __init__(self, name: str, description=None):
-        super().__init__(name, description=description)
-        self.name = "property/read"
-
-
-class WriteProperty(Property):
-    def __init__(self, name: str, description=None):
-        super().__init__(name, description=description)
-        self.name = "property/write"
+    def __init__(self, name: str, access: str, description=None):
+        super(Property, self).__init__(name, description=description)
+        self.name = access
```

Comment:
`self.access = access`?

Reply:
While printing a feature, the `__str__` method of the `Feature` class is called https://github.com/mandiant/capa/blob/master/capa/features/common.py#L131 that prints the `name` of the feature. So, I am setting the name of the `Property` feature as either `property/read` or `property/write` with `self.name = access` to avoid having to override the method. I understand that it is not a clean solution, but it currently works without having to make much changes/handle special conditions in other parts of the code. If we were to include the `access` attribute in the `Feature` class, then it would eliminate this need.